modern/src/Actions.ts

```diff
@@ -34,8 +34,33 @@ export function gotSkinBlob(blob: Blob, store: ModernStore) {
   };
 }
 
+async function unloadSkin(makiTree) {
+  await Utils.asyncTreeFlatMap(makiTree, async (node: MakiObject) => {
+    switch (node.name) {
+      case "groupdef": {
+        // removes groupdefs from consideration (only run scripts when actually referenced by group)
+        return {};
```

Comment:
Because we dont want to consider anything inside a `groupdef`. I guess it doesn't actually matter since the `script`s inside a `groupdef` wouldn't have a `system` anyway.